es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
 <br>
 
 **33. Regularization ― The regularization procedure aims at avoiding the model to overfit the data and thus deals with high variance issues. The following table sums up the different types of commonly used regularization techniques:**
 
-&#10230;
+&#10230; Regularización - el procedimiento de regularización tiene como objetivo evitar que el modelo se sobreajuste a los datos y, por lo tanto, resuelve los problemas de alta varianza. La siguiente tabla resume los diferentes tipos de técnicas de regularización comúnmente utilizadas:
 
 <br>
 
 **34. [Shrinks coefficients to 0, Good for variable selection, Makes coefficients smaller, Tradeoff between variable selection and small coefficients]**
 
-&#10230;
+&#10230; [Reduce los coeficientes a 0, Bueno para la selección de variables, Hace que los coeficientes sean más pequeños, Compensación entre la selección de variables y los coeficientes pequeños]
 
 <br>
 
 **35. Diagnostics**
 
-&#10230;
+&#10230; Diagnóstico
 
 <br>
 
 **36. Bias ― The bias of a model is the difference between the expected prediction and the correct model that we try to predict for given data points.**
 
-&#10230;
+&#10230; Sesgo - el sesgo de un modelo es la diferencia entre la predicción esperada y el modelo correcto que tratamos de predecir para determinados puntos de datos.
 
 <br>
 
 **37. Variance ― The variance of a model is the variability of the model prediction for given data points.**
 
-&#10230;
+&#10230; Varianza - la varianza de un modelo es la variabilidad de la predicción del modelo para puntos de datos dados.
 
 <br>
 
 **38. Bias/variance tradeoff ― The simpler the model, the higher the bias, and the more complex the model, the higher the variance.**
 
-&#10230;
+&#10230; Corrección de sesgo/varianza - cuanto más simple es el modelo, mayor es el sesgo, y cuanto más complejo es el modelo, mayor es la varianza.
 
 <br>
 
 **39. [Symptoms, Regression illustration, classification illustration, deep learning illustration, possible remedies]**
 
-&#10230;
+&#10230; [Síntomas, ejemplo de regresión, ejemplo de clasificación, ejemplo de aprendizaje profundo, posibles soluciones]
 
 <br>
 
 **40. [High training error, Training error close to test error, High bias, Training error slightly lower than test error, Very low training error, Training error much lower than test error, High variance]**
 
-&#10230;
+&#10230; [Error de entrenamiento alto, Error de entrenamiento cercano al error de prueba, Sesgo alto, Error de entrenamiento ligeramente inferior al error de prueba, Error de entrenamiento muy bajo, Error de entrenamiento mucho más bajo que el error de prueba, Varianza alta]
 
 <br>
 
 **41. [Complexify model, Add more features, Train longer, Perform regularization, Get more data]**
 
-&#10230;
+&#10230; [Complementar el modelo, agregar más funciones, entrenar más tiempo, realizar la regularización, obtener más datos]
```

Comment:
I think "Complexify model" could be "Incrementar la complejidad del modelo", "Volver más complejo el modelo".